serverless-plugin/default-config.yaml

```diff
@@ -101,6 +101,14 @@ alarms:
     NumberOfNotificationsFailed:
       Statistic: Sum 
       Threshold: 1
+  Events:
```

Comment:
These should also be added to the README where we show the default configuration